frontend/src/lib/k8s/service.ts

```diff
@@ -101,7 +101,12 @@ class Service extends KubeObject<KubeService> {
   }
 
   getPorts() {
-    return this.spec?.ports?.map(port => port.port);
+    return this.spec?.ports?.map(({ port, nodePort, targetPort, protocol }) => {
```

Comment:
We can't change getPorts method like this because some plugins may use it and this is a breaking change.
I suggest using a separate method like `getFormattedPorts` or something like that